spring-cloud-contract-verifier/src/main/java/org/springframework/cloud/contract/verifier/messaging/amqp/SpringAmqpStubMessages.java

```diff
@@ -85,20 +86,37 @@ public SpringAmqpStubMessages(RabbitTemplate rabbitTemplate, MessageListenerAcce
 	}
 
 	@Override
-	public <T> void send(T payload, Map<String, Object> headers, String destination, YamlContract contract) {
+	public <T> void send(T payload, Map<String, Object> messageHeaders, String destination, YamlContract contract) {
```

Comment:
Yeah we can add `@Nullable` for message headers.

Reply:
will do for the interface

Reply:
But wait, I have just checked another implementation - `JmsStubMessages`, it does not expect null headers in its `send() `method, because it tries to get `entrySet()`.

Reply:
So if you may, can you do null checks there too?

Reply:
I would rather create a separate issue for that